templates/zerver/login.html

```diff
@@ -67,11 +67,12 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
                                 </label>
                             </div>
 
-                            <div class="input-box no-validation">
-                                <input id="id_password" name="password" class="required" type="password"
+                            <div class="input-box no-validation password-div">
+                                <input id="id_password" name="password" class="required" type="password" autocomplete="off"
                                   {% if email %} autofocus {% endif %}
                                   required />
                                 <label for="id_password" class="control-label">{{ _('Password') }}</label>
+                                <i class="fa fa-eye-slash password_visibility_toggle" role="button" aria-label="{{ _('Toggle password visibility') }}"></i>
```

Comment:
I think we need to set `title` here, or perhaps setup TippyJS -- otherwise hovering the element does not display the label.  @amanagr FYI in case you have advice.